sdk/storage/storage-blob/gulpfile.js

```diff
@@ -6,11 +6,7 @@ const zipFileName = `azurestoragejs.blob-${version}.zip`;
 
 gulp.task("zip", function(callback) {
   gulp
-    .src([
-      "browser/azure-storage.blob.js",
-      "browser/azure-storage.blob.min.js",
-      "browser/*.txt"
-    ])
+    .src(["browser/index.js", "browser/*.txt"])
```

Comment:
I believe we still want the minified version. The template uses the `uglify` rollup plugin to do that.

Reply:
[ Offline discussion with Jeremy ]
- As per the template, the generated file is - `browser/index.js` (minified version)
- This PR uses `rollup-plugin-uglify` to generate the minified version ("browser/index.js") just like the template does
- Not sure if we should generate both the files
["browser/azure-storage.blob.js", "browser/azure-storage.blob.min.js" ]
 the way storage-blob previously does
- Still have to decide on the names for generated outputs!

Reply:
My thoughts here: the standalone unminified version seems not important for npm package scenarios, but is possibly important when downloading a .zip release from e.g. GitHub. If we go the GH release route, we should have an unminified version, but otherwise I'm fine leaving it out. I'll also note that the unminified code will likely not be super nice to debug as it will still be bundled and have a bunch of "plumbing" goop.

In terms of file names we discussed on teams, how about we go with the following rule: the bundle file name is the package name with @ removed, / replaced with -, and suffixed with `.min` if minified. So `azure-storage-blob.min.js`.

Reply:
"Modify template to generate both minified and unminified versions of src code through rollup and update the names of outputs"
Issue to track  - https://github.com/Azure/azure-sdk-for-js/issues/2416

Reply:
Both js and min.js are needed in the generated zip bundle to be published.

Reply:
Yes @XiaoningLiu, we are doing it now.
Now, the generated zip bundle contains both "azure-storage-blob.min.js" and "azure-storage-blob.js"